.github/workflows/test.yml

```diff
@@ -256,7 +256,7 @@ jobs:
           sudo apt update -y
           sudo apt install -y --no-install-recommends git g++ cmake ninja-build llvm-15-dev zlib1g-dev libglew-dev flex bison libfl-dev libboost-thread-dev libboost-filesystem-dev nvidia-cuda-toolkit-gcc
       - name: Cache gpuocelot
-        if: matrix.backend == 'cuda' || matrix.backend == 'ptx'|| matrix.backend == 'triton'
+        if: ${{false}} # matrix.backend == 'cuda' || matrix.backend == 'ptx' #|| matrix.backend == 'triton'
```

Comment:
We need this cache or it's slow, why is it broken?

Reply:
I've restored the cache - remainder as per our discussion in discord::triton_backend. Cache is probably broken on master and it needs to be cleared before merging this or it will time out again. I think if you remove the cache and merge a commit to this pr the tests should run on master